src/domain/home/Home.tsx

```diff
@@ -22,6 +23,7 @@ function Home(props: Props) {
           </div>
         </div>
       </div>
+      <button onClick={authenticate}>Authenticate using tunnistamo</button>
```

Comment:
Shouldn't this text be included in the translations files?

Reply:
That will happen when we add the actual login menu in the header..